bin/publish.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
```

Comment:
There should be a `set -e` at the top of the shell file (causes all errors to be fatal)